src/cljc/spy/test.cljc

```diff
@@ -96,46 +104,59 @@
                         ~`expected-not-called
                         ~`actual-not-called))
 
+     (defmacro assert-call-matching
+       [msg form report-fn]
+       `(assert-called ~msg
+                       ~form
+                       ~report-fn
+                       (fn [~'& ~'args]
+                         (expected-call-matching '~(last form)))
```

Comment:
This passes the quoted form to `expected-call-matching` so it can be used in the expected message.